src/datasets/other/cdn/intro.md

```diff
@@ -0,0 +1,16 @@
+There are two [Content delivery networks (CDNs)](https://en.wikipedia.org/wiki/Content_delivery_network) Mozilla utilizes for delivery of its products: AWS Cloudfront, and Akamai.
+
+* Stub Installers are downloaded from download-installer.cdn.mozilla.net (Cloudfront)
+* Full Installer downloads initiated by stub for V60+ download from download-installer.cdn.mozilla.net (Cloudfront)
+* Full Installers initiated by stub for pre-V60 download from download.cdn.mozilla.net (Cloudfront)
+* FTP hosted by Cloudfront 
+
+| FTP                               | http://ftp.stage.mozaws.net        | Cloudfront          |
+|-----------------------------------|------------------------------------|---------------------|
```

Comment:
This is a table header; should it be? Is this table redundant with the text? I think you'd just need one.